### PR TITLE
Change page structure to mimic 404 page

### PR DESCRIPTION
**Pull request by NatasaKralj**
Fix for Search page, the look and spacing of elements is taken from 404 page.

### CODE REVIEW

File: themes/mendix/layouts/search/single.html [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
-            <div id="tags" class="search_filters_block"></div>
+      <div class="row mx__main-content">
+        <!-- CONTENT -->
+        <div class="col-md-9 col-xs-12 col-md-push-3 mx__main">
+          <div class="mx__page mx__page--frontpage">
+            <div class="row mx__page__header">
+              <div class="col-md-12 col-xs-12">
+                <header class="mx__page__header__head">
+                  <h1 class="mx__page__header__title mx__page__header__title--frontpage">What can we help you with?</h1>
+                  <h2 class="mx__page__header__subtitle">Use the search bar at the top left of the page.</h2>
```
Comment (MarkvanMents) [inline themes/mendix/layouts/search/single.html:15]:
```suggestion
                  <h2 class="mx__page__header__subtitle">Use the search bar at the top right of the page.</h2>
```
I think?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Looks great - I've added one comment.
Still not sure quite how to get to this page as when I run locally and go to /search I get the suggestions page. It is only if I do /search/ that I get this page.
But we have at least solved the issue and learned something new about the site: so it is all progress.
If you are happy with my suggestion, just commit it and then let me know that I can merge the PR.

**Comment by NatasaKralj** [comment]
@MarkvanMents I accepted your suggestion, good to merge PR.

**Review by MarkvanMents — APPROVED** [review]
Great.
Hope you can be given access to merge your own PRs next time.

**Pull request merged**